cmd/schemagen/schemagen.go

```diff
@@ -26,6 +27,7 @@ var (
 	flagOutput   = cmd.String("output", "models", "the name of the folder to output to")
 	flagUser     = cmd.String("user", "", "user for password authentication")
 	flagPassword = cmd.String("password", "", "password for password authentication")
+	flagTimeout  = cmd.Int("timeout", 5, "connection timeout for the connection to the database ( in seconds )")
```

Comment:
I think default should be in line with `gocql` default value, which is `11` (https://github.com/gocql/gocql/blob/master/cluster.go#L262)